modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeDefinitionServiceImpl.java

```diff
@@ -106,14 +106,15 @@ public int getListTypeDefinitionsCount() {
 
 	@Override
 	public ListTypeDefinition updateListTypeDefinition(
-			long listTypeDefinitionId, Map<Locale, String> nameMap)
+			long listTypeDefinitionId, Map<Locale, String> nameMap,
+			String externalReferenceCode)
```

Comment:
@murilostodolni @mateussgn `String externalReferenceCode` must come before `listTypeDefinitionId`, following the same order from the `service.xml`